formstack-consents/src/main/scala/com/gu/identity/formstackconsents/Lambda.scala

```diff
@@ -52,23 +52,43 @@ object Lambda extends StrictLogging {
   }
 }
 
-case class FormstackSubmission(formId: String, emailAddress: String, handshakeKey: String)
+case class FormstackSubmission(
+  formId: String,
+  emailAddress: String,
+  handshakeKey: String,
+  opt_in: Option[Boolean] = None // if required, request the formstack checkbox hidden label be "opt_in"
+)
 
 object FormstackSubmission {
 
   private def getEmailField(cursor: HCursor): Decoder.Result[String] = {
     cursor.downField("email_address").as[String]
       .orElse(cursor.downField("your_email_address").as[String])
+        .orElse(cursor.downField("email").as[String])
+  }
+
+  // Form submissions of type MarketingConsent require the user to 'opt in' to consents, not just submit.
+  // While we can conditionally trigger lambda webhook within Formstack, these settings are not exclusively
+  // under our control so we perform an additional checks in code to ensure we are never collecting consent and user data inappropriately.
+  // Any forms with required opt ins should be added in Newsletter.scala. Request the formstack checkbox hidden label be "opt_in"
+  private def getConsentOptInField(cursor: HCursor): Option[Boolean] = {
```

Comment:
@vlbee This is minor but you should use `/**` for multiline comment rather than `//`